Cylenium.Tests/DriverTests.cs

```diff
@@ -37,5 +38,17 @@ public void Set_and_get_the_window_size()
             var expectedSize = cy.WindowSize(width: 800, height: 600);
             Assert.AreEqual(cy.WindowSize(), expectedSize);
         }
+
+        [Test]
+        [Category("Driver")]
+        public void Count_number_of_tabs()
+        {
+            var wait = cy.Wait();
+            string test_url = "https://www.qap.dev/";
```

Comment:
You aren't using `var wait` so you can get rid of this line.

Also, if you want, you can just include `https://www.qap.dev/` in your `ExecuteScript` line instead of holding it in a variable.